charts/nd-common/templates/_datadog.tpl

```diff
@@ -28,6 +28,24 @@ ad.datadoghq.com/{{ .Chart.Name }}.instances: |-
   ]
 {{- end }}
 
+{{/*
+This is datalog logging configuration. We take the .Values.scrapeLogs and
+.Values.scrapeLogsProcessingRules map and convert into list of objects converted
+into json supported by datadog config. If source and service tag values not provided
+we add default values to it.
+*/}}
+
+{{- if and .Values.datadog.enabled .Values.datadog.scrapeLogs.enabled }}
+ad.datadoghq.com/{{ .Release.Name }}.logs: |-
+  [
+    {
+      "source": {{- default .Release.Name .Values.datadog.scrapeLogs.source | toJson }},
+      "service": {{- default .Release.Name .Values.datadog.service | toJson }},
+      "log_processing_rules": {{- .Values.datadog.scrapeLogsProcessingRules | toJson }}
```

Comment:
For the users sake, it's probably worth running this through `tpl` as well...

Reply:
done